docs/MQTT-streamingsource.md

```diff
@@ -0,0 +1,116 @@
+MQTT Streaming Source
+=====================
+
+[![Join CDAP community](https://cdap-users.herokuapp.com/badge.svg?t=mqtt)](https://cdap-users.herokuapp.com)
+[![Build Status](https://travis-ci.org/hydrator/mqtt.svg?branch=develop)](https://travis-ci.org/hydrator/mqtt)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+![cdap-transform](https://cdap-users.herokuapp.com/assets/cdap-transform.svg)
+
+This MQTT Streaming Source allows you to subscribe to a specific topic with an MQTT broker as a
+client for messages in a streaming context.
+
```

Comment:
What about:
```
The MQTT Streaming Source allows you to subscribe to a specific topic with an MQTT broker as a
client for messages in a streaming context.
```
The sentence is still running-on; if you could break into two or improve it, that'd be good.